worlds/ladx/Tracker.py

```diff
@@ -116,16 +117,17 @@ def __init__(self, gameboy):
             if check_id == '0x2A3':
                 self.start_check = check
             self.all_checks.append(check)
-        self.remaining_checks = [check for check in self.all_checks]
+            self.meta_to_check_dict[check.id] = check
+        self.remaining_checks = {check.address: check for check in self.all_checks}
         self.gameboy.set_cache_limits(
             lowest_check, highest_check - lowest_check + 1)
 
     def has_start_item(self):
-        return self.start_check not in self.remaining_checks
+        return self.start_check.address not in self.remaining_checks.keys()
 
     async def readChecks(self, cb):
         new_checks = []
-        for check in self.remaining_checks:
+        for check in list(self.remaining_checks.values()):

```

Comment:
this list() cast is probably unneccessary

Reply:
Removed.